src/main/java/dev/jbang/Script.java

```diff
@@ -629,4 +629,8 @@ public String getJavaAgentOption() {
 	public List<KeyValue> getAgentOptions() {
 		return agentOptions;
 	}
+
+	public String getScript() {
```

Comment:
not fan of exposing this - should be accessing what we need from this instance so we can agressively cache results rather than rerun them.

Reply:
So my last changes were made to be possible to search for `//SOURCES` in multiple files.
`Script.collectSources` only works for the main file (entry point), that's why I created Util.collectSources(String content) so it can work recursively. But then I needed to pass Script.script to this new method, that's why I "exposed" it. =)

But now I realized that for the initial file I can still call `Script.collectSources`, so I removed `getScript` and now it's not exposed anymore. :]

Reply:
I've made a commit that moves the static methods back to Util. 

I'm still a bit eery that we are collecting sources by iterating raw source strings rather than build up list of Script (or some other class) so we at least could attach other info to them - like which deps they have.

But I think we can do that in a follow up when we need it.

Reply:
Store in some kind of data structure maybe it is a better approach. Script could have a list of ScriptResource or some new class (Source?)

I thought about that but it would be a much complex change, touching many places ...

But the good thing is that I think this change is not really needed now, because once we get all `//SOURCES` recursively, their `Path`s will be stored in `Script.resolvedSources`. So we only need to loop through them to get `//DEPS` and `//FILES`